### DISCUSSION TITLE
[addon-table] How to know the column which is sorted by?

### DISCUSSION THREAD

**Discussion by spicy-tomato** [taiga-family/taiga-ui#2301] [Q&A] [answered]
Example, when I click column 'Number', something should emit 'numberReality'. Tried to emit (sorterChange) of tuiTable, but always receive default comparator

![image](https://user-images.githubusercontent.com/62208789/183335446-2ecf0603-e931-47f6-bafd-c157b7c1dd02.png)
![image](https://user-images.githubusercontent.com/62208789/183335500-19fabf84-0742-4973-b6ab-291f69d0586c.png)

**Comment by waterplea** [comment, marked as answer]
Hey there. You can use `tuiSortBy` directive to operate with column names. Here's an example:
https://taiga-ui.dev/components/table#server

**Reply by spicy-tomato** [reply]
Thank you
